src/routes/Root/index.tsx

```diff
@@ -61,7 +61,7 @@ const Root = () => {
 		fetch(CoreURL + "getRecipeDescription", {"method": "POST", "body": JSON.stringify({recipe})})
 		.then(r => r.json())
 		.then(desc => {
-			recipeDescriptions[recipe] = desc["description"] ?? "";
+			recipeDescriptions[recipe] = desc["description"] ?? "Unknown Package";
```

Comment:
Would be better if this explicitly also said " (due to being from a module)" if it's a module env.

Reply:
All unknown packages should be from modules, so I've changed the text to `Unknown Module Package`.